Solutions/GitHub/Hunting Queries/Repository Permission Switched to Public.yaml

```diff
@@ -0,0 +1,16 @@
+id: a6e2afd3-559c-4e88-a693-39c1f6789ef1
+name: GitHub Repo switched from private to public
+description: |
+  'This hunting query identifies GitHub activites where a repo was changed from private to public that may be a sign of compromise.'
+requiredDataConnectors: []
```

Comment:
Please add the connectorid and datatype in this format
requiredDataConnectors: -
connectorId: connector id
dataTypes: - name of the parser which we use in the query

Thanks,
Aditi Maheshwari

Reply:
Added